app/views/posts/_create_comments.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_with url: "/posts/#{@post.id}/comments", scope: :comments do |f| %>
```

Comment:
- [ ] There should be a form in the views/comments/new.html.erb file which accepts input for creating comments. When I explore your views I was not able to find it. Kindly add it as well :)

Reply:
I used the form from the Posts page to avoid repetition (DRY) and to not refresh the page to type a new comment, and it is not a part of the requirement to write forms in every code file!